srcpkgs/python-tqdm/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'python-tqdm'
+pkgname=python-tqdm
+version=4.28.1
+revision=1
+noarch=yes
+wrksrc="tqdm-${version}"
+build_style=python-module
+pycompile_module="tqdm"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
+short_desc="Fast, extensible progress bar for Python and CLI (Python2)"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="MPL-2.0, MIT"
+homepage="https://tqdm.github.io/"
+distfiles="https://github.com/tqdm/tqdm/archive/v${version}.tar.gz"
+checksum=3a405bcf0b4f30c72d2f7d10f7574d952b837abb5777dc2ed0f0a57a8943d171
+alternatives="
+ tqdm:tqdm:/usr/bin/tqdm2
+ tqdm:tqdm.1:/usr/share/man/man1/tqdm2.1"
+
+post_install() {
```

Comment:
There is a manpage in ${py3_sitelib} and ${py2_sitelib}

Reply:
Not anymore